tools/clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -782,6 +784,13 @@ void clang::EmitBackendOutput(DiagnosticsEngine &Diags,
   } catch (const ::hlsl::Exception &hlslException) {
     Diags.Report(Diags.getCustomDiagID(DiagnosticsEngine::Error, "%0\n"))
         << StringRef(hlslException.what());
+#if defined(DXC_CODEGEN_EXCEPTIONS_TRAP)
+    // llvm::errs() doesn't work in release builds on Linux.
```

Comment:
... in libdxcompiler.so which is mainly where I want to use it.

Reply:
Ugh! I had a patch somewhere that fixed that, and I don't remember what the issue was. It was something _really_ silly.